test/integration/Main.hs

```diff
@@ -7,10 +7,13 @@ import Control.Concurrent.MVar
 
 import Data.Aeson
     ( Value )
+import Data.ByteString.Lazy
+    ( ByteString )
 import Data.Text
     ( Text )
 import Network.HTTP.Client
-    ( Manager, defaultManagerSettings, newManager )
+    ( Manager, Request, Response, defaultManagerSettings, newManager )
+import Network.HTTP.Types.Status
 import Prelude
```

Comment:
`Prelude` at the top, then newline, then explicit imports, then newline, then qualified imports (cf coding standards)